daemons/controld/controld_fencing.c

```diff
@@ -961,11 +962,12 @@ controld_execute_fence_action(pcmk__graph_t *graph,
 bool
 controld_verify_stonith_watchdog_timeout(const char *value)
 {
+    const char *our_nodename = controld_globals.our_nodename;
     gboolean rv = TRUE;
 
     if (stonith_api && (stonith_api->state != stonith_disconnected) &&
         stonith__watchdog_fencing_enabled_for_node_api(stonith_api,
-                                                       fsa_our_uname)) {
+                                                       our_nodename)) {
```

Comment:
per the guidelines it's fine to just indent the arguments 4 spaces when it gets this ridiculous

Reply:
I don't remember seeing that in the guidelines, and I don't see it now while skimming them. Can you point to a section? The closest thing I see is "Lines should be no longer than 80 characters unless limiting line length hurts readability," but that refers to lengthening lines, not beginning them closer to the left.

With that said, I'd rather just shorten the function name in this case. It's internal and very long.

Reply:
I completely forgot I dropped that in ac1add503 ... I think I was thinking that we should just shorten the function name in such cases :)

Reply:
In any case it's fine to leave this as you have it